src/main/java/uk/gov/pay/adminusers/app/healthchecks/MigrateToInitialDbState.java

```diff
@@ -38,7 +38,7 @@ protected void run(Bootstrap<AdminUsersConfig> bootstrap, Namespace namespace, A
             resultSet.close();
             statement.close();
         } catch (SQLException e) {
-            logger.error("Error during initial DB setup", e.getMessage());
+            logger.error("Error during initial DB setup", e);
```

Comment:
Putting the whole exception tends to toss each line of the stack trace into Sumo Logic as a separate log message, so it’s possible `e.getMessage()` was considered. But, meh, change it, it’s not like we expect this error to ever actually happen and a stack trace would probably be useful if it did.

Reply:
gah, had to rebase